### PR TITLE
corner plot script

### PR DESCRIPTION
**Pull request by sahiljhawar**
Script to generate corner plots. Single posterior sample or multiple posterior samples. In case of multiple posterior datasets all posterior samples should have identical parameters.

### CODE REVIEW

**Comment by mcoughlin** [comment]
@tsunhopang can you please review and merge if happy?

File: tools/corner_plot.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
+import corner
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import argparse
+import os
+import seaborn as sns
+import matplotlib.patches as mpatches
+import re
```
Comment (tsunhopang) [inline tools/corner_plot.py:9]: could u add the `matplotlib.use("agg")` so that it also work on mac

**Comment by mcoughlin** [comment]
@tsunhopang Please rereview.

**Review by tsunhopang — APPROVED** [review]
lgtm

**Pull request merged**